vscode/src/webview/webview.tsx

```diff
@@ -47,24 +49,26 @@ type EstimatesState = {
 
 type CircuitState = {
   viewType: "circuit";
+  panelId: string;
   props: CircuitProps;
 };
 
 type DocumentationState = {
   viewType: "documentation";
+  panelId: string;
```

Comment:
Do you really need to pass in `panelId` for Documentation and Estimates, since they don't actually have distinct panel IDs?

Reply:
removed